README.md

```diff
@@ -4,43 +4,42 @@
 
 ## WARNING!
 
-This is _pre alpha_ software with super obvious bugs, rough edges etc. In the spirit of "release early and release often", I am posting the code. This project is far from done but I prefer to get the code out there to be used by whomever. Disagree with the project? Fork it :)
+This is _pre-alpha_ software with super obvious bugs, rough edges etc. In the spirit of "release early and release often", I am posting the code. This project is far from done but I prefer to get the code out there to be used by whomever. Disagree with the project? Fork it :)
 
-## Obligatory Screenshot
 ![KotCity Screenshot](screenshot.jpg?raw=true "Screenshot of the game's UI and an example city")
 
-## Gimme the Software!
+## New in this version
 
-A build for Windows, macOS and Linux is available at https://github.com/kotcity/kotcity/releases/tag/0.45
+95% of these are from my main man @sabieber, who is the KotCity hero.
 
-Java 8+ is required. On Windows it will look for JRE and bring you to download page if you don't have it.
+See our roadmap at https://github.com/kotcity/kotcity/wiki/Roadmap
 
-(note, on Ubuntu do apt-get install openjfx)
+* Displaying population.
+* Buildings construct a little more often.
+* Tweaked required kots (equivalent to *sims*) for industrial plant.
+* Minimal map size reduced (you can start a 32*32 map).
+* Right click and drag to pan map.
+* Update cursor on map pan.
+* Dynamic framerate adjustment during panning & zooming.
+* Zoom to mouse position on zoom.
 
-## New in this version
+## What is KotCity?
```

Comment:
Why all the moving around again? I think having the link to the release above all the text is a lot better